source/common/network/raw_buffer_socket.h

```diff
@@ -21,6 +21,7 @@ class RawBufferSocket : public TransportSocket, protected Logger::Loggable<Logge
   IoResult doRead(Buffer::Instance& buffer) override;
   IoResult doWrite(Buffer::Instance& buffer, bool end_stream) override;
   Ssl::ConnectionInfoConstSharedPtr ssl() const override { return nullptr; }
+  bool startSecureTransport() override { return false; }
```

Comment:
Test coverage for this line seems to be flaky.  Some runs seem to be hitting it but some aren't.

hit: https://storage.googleapis.com/envoy-pr/2de21bf/coverage/source/common/network/raw_buffer_socket.h.gcov.html
not hit: https://storage.googleapis.com/envoy-pr/48c0687/coverage/source/common/network/raw_buffer_socket.h.gcov.html

Reply:
Thanks @antoniovicente. I will investigate it.